classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom.push(self) unless student.classroom.include?(self)
+  end
```

Comment:
- [ ] The `Classroom` has many `Students` while the `Student` belongs to a `Classroom`. To mirror this, the `classroom` attribute which you have in the `Student` class should be set to self. It is an object and not an array.

`student.classroom = self`

Reply:
Nice catch @Gambit142!
I missed that `student.classroom` is not an array.